core/example/longrange/CMakeLists.txt

```diff
@@ -1,9 +1,11 @@
-if(Cabana_ENABLE_MPI)
-  add_executable(Direct directsum_example.cpp)
-  target_link_libraries(Direct cabanacore)
-  add_executable(Ewald ewaldsum_example.cpp)
-  target_link_libraries(Ewald cabanacore)
+if(NOT Cabana_ENABLE_MPI)
+  return()
```

Comment:
Come on :@

Reply:
Don't include the subdirectory then

Reply:
:-)

Reply:
We do the same in a couple of places:
```
git grep 'return()' -- '*CMakeLists.txt'
core/example/CMakeLists.txt:  return()
core/fortran-example/tutorial/05_parallel_for/CMakeLists.txt:    return()
core/fortran-example/tutorial/06_kokkos_parallel_for/CMakeLists.txt:  return()
core/performance_test/peakflops/CMakeLists.txt:  return()
core/performance_test/peakflops/CMakeLists.txt:    return()
core/unit_test/CMakeLists.txt:  return()
```

Reply:
I prefer this way as it localizes the restriction more, but I am happy to exclude the subdir.

Reply:
Fine